src/lib/viewers/image/ImageControls.tsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import AnnotationsControls, { Props as AnnotationsControlsProps } from '../controls/annotations';
 import ControlsBar from '../controls/controls-bar';
+import DrawingControls from '../controls/annotations/DrawingControls';
 import FullscreenToggle, { Props as FullscreenToggleProps } from '../controls/fullscreen';
 import RotateControl, { Props as RotateControlProps } from '../controls/rotate';
 import ZoomControls, { Props as ZoomControlsProps } from '../controls/zoom';
 
-export type Props = AnnotationsControlsProps & FullscreenToggleProps & RotateControlProps & ZoomControlsProps;
+export type Props = AnnotationsControlsProps &
+    FullscreenToggleProps &
+    RotateControlProps &
+    ZoomControlsProps & {
+        onAnnotationColorChange: (color: string) => void;
```

Comment:
If we add `DrawingControlsProps` to our union, we can remove this extra property.